.github/workflows/build-test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     uses: playframework/.github/.github/workflows/cmd.yml@v4
     with:
       java: 8
-      scala: 2.11.x, 2.12.x
```

Comment:
Why?

Reply:
Good question: Why? You actually made this commit.
I was wondering this myself.